ocaml/sdk-gen/go/gen_go_helper.ml

```diff
@@ -0,0 +1,199 @@
+(* Copyright (c) Cloud Software Group, Inc.
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU Lesser General Public License as published
+   by the Free Software Foundation; version 2.1 only. with the special
+   exception on linking described in file LICENSE.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Lesser General Public License for more details.
+*)
+
+(* Generator of Go bindings from the datamodel *)
+
+open Datamodel_types
+open CommonFunctions
+module Types = Datamodel_utils.Types
+
+let templates_dir = "templates"
+
+let ( // ) = Filename.concat
+
+let snake_to_camel (s : string) : string =
+  Astring.String.cuts ~sep:"_" s
+  |> List.map (fun s -> Astring.String.cuts ~sep:"-" s)
+  |> List.concat
+  |> List.map String.capitalize_ascii
+  |> String.concat ""
+
+let render_template template_file json =
```

Comment:
See above for my comment to add a `?(nl=false)` parameter to add a final `\n` to the output.